README.md

```diff
@@ -115,10 +115,24 @@ Apps should have their directory names prefixed with `app-`, e.g. `/project` bec
 
 ## Production deployment
 
-Deploys to production are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image is created which installs and builds the `lib-` packages, and that's used as a base image for creating `app-` images, which are then deployed to Kubernetes.
+Deploys are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image is created which installs and builds the `lib-` packages, and that's used as a base image for creating `app-` images, which are then deployed to Kubernetes.
 
 More information is available in [ADR 12](docs/arch/adr-12.md).
 
+Merges to master deploy to a staging instance that uses Panoptes production. This is used for manual end to end behavior testing for new code and design reviews. Deployments to a production are triggered by setting a `production-release` git tag. This can either be done using the git CLI or using lita on slack. 
```

Comment:
```suggestion
Merges to master deploy to a staging instance that uses Panoptes production. This is used for manual end-to-end behavior testing for new code and design reviews. Deployments to production are triggered by setting a `production-release` git tag. This can either be done using the git CLI or using lita on slack. 
```

Reply:
I _think_ deploying to production involves tagging a commit with `production-release` and running `lita deploy front-end-monorepo`. I'm not sure that pushing the tag on its own will trigger a deploy.

Reply:
the lita deploy commands commits the git tag for you: https://github.com/zooniverse/lita/blob/master/handlers/deployment.rb#L63